package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Weeb bot",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "nodemon index.js",
```

Comment:
What are your thoughts on making `nodemon` a developer-dependency instead? We could make another npm script like `npm run dev` that will invoke `nodemon index.js` during development

Reply:
Main reason is because nodemon is introducing a bunch of packages that doesn't seem necessary for production environment

Reply:
👍 for making this a dev dependency.